components/brave_wallet/browser/eth_tx_service.cc

```diff
@@ -18,6 +18,7 @@
 #include "brave/components/brave_wallet/browser/asset_ratio_service.h"
 #include "brave/components/brave_wallet/browser/brave_wallet_constants.h"
 #include "brave/components/brave_wallet/browser/brave_wallet_prefs.h"
+#include "brave/components/brave_wallet/browser/brave_wallet_utils.h"
```

Comment:
left over?

Reply:
Yes, removed in https://github.com/brave/brave-core/pull/11929/commits/eb2a3a6dc9ae408c0d75f6512c5e0298b57153fa